### PR TITLE
120 Limited: Add Quiz

### PR DESCRIPTION
**Pull request by leogermond**
- 120: syntax quiz
- 120: constructor quiz
- 120: add dfn
- 120: quiz on operators
- 120: remove duplicated quiz
- 120: fix quiz name
- 120: limited constructor syntax quiz
- 120: add limited private quiz
- 120: quiz typo

### CODE REVIEW

File: courses/fundamentals_of_ada/quiz/limited_constructor_syntax/prj.gpr [hunk 1]
```diff
@@ -0,0 +1,13 @@
+project Prj is
```
Comment (frank-at-adacore) [inline courses/fundamentals_of_ada/quiz/limited_constructor_syntax/prj.gpr:1]: Same comment as previous reviews: Common project file, drop the ADC file

File: courses/fundamentals_of_ada/quiz/limited_extended_return/quiz.rst [hunk 2]
```diff
@@ -0,0 +1,18 @@
+.. code:: Ada
```
Comment (frank-at-adacore) [inline courses/fundamentals_of_ada/quiz/limited_extended_return/quiz.rst:1]: This is the basically the same as the constructor quiz just adding and "extended return" option. Change this to show bad extended return constructs

File: courses/fundamentals_of_ada/quiz/limited_private/quiz.rst [hunk 3]
```diff
@@ -0,0 +1,28 @@
+.. code:: Ada
+
+    package P is
+       type Priv is private;
+    private
+       type Lim is limited null record;
+    end P;
+
+Which of the following piece(s) of code is(are) legal?
```
Comment (frank-at-adacore) [inline courses/fundamentals_of_ada/quiz/limited_private/quiz.rst:9]: This is a little confusing. Put a comment or something after "type Lim" to show where the answers would actually appear in the package

**Review by frank-at-adacore — DISMISSED** [review]

File: courses/fundamentals_of_ada/quiz/limited_extended_return/quiz.rst (hunk 2 above)
Comment (leogermond) [reply courses/fundamentals_of_ada/quiz/limited_extended_return/quiz.rst:1]: That's true!

**Review by frank-at-adacore — DISMISSED** [review]

**Review by rchoch — APPROVED** [review]

File: courses/fundamentals_of_ada/120_limited_types.rst [hunk 4]
```diff
@@ -216,6 +216,18 @@ Composites with Limited Types
 Quiz
```
[the remark this answers is not in the document]
Comment (leogermond) [reply courses/fundamentals_of_ada/120_limited_types.rst:216]: Ah indeed, thanks for catching that up!

File: courses/fundamentals_of_ada/quiz/limited_private/quiz.rst [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+   |    ``F : A;``
+   | ``end record;``
+D. | :answermono:`type Acc is access Lim;`
+   | :answermono:`type Priv is record`
+   |    :answermono:`F : Acc;`
+   | :answermono:`end record;`
+
+.. container:: animate
+
+    A. ``E`` has limited type, partial view of Priv must be :ada:`private limited`
```
Comment (frank-at-adacore) [inline courses/fundamentals_of_ada/quiz/limited_private/quiz.rst:29]: This is the same reasoning for C (which is correctly flagged as a wrong answer)

**Review by frank-at-adacore — CHANGES_REQUESTED** [review]

**Review by frank-at-adacore — APPROVED** [review]

**Pull request merged**